apps/dapp/src/hooks/core/use-deposit-to-vault.tsx

```diff
@@ -28,8 +29,10 @@ export const useDepositToVault = (vaultContractAddress: string, onSuccess?: Call
   const [getZappedAssetValue] = useGetZappedAssetValue();
 
   const { openNotification } = useNotification();
+  const { captureEvent } = AnalyticsService.getInstance();
 
   const handler = async (ticker: TICKER_SYMBOL, amount: string, useFaith = false) => {
+    captureEvent('[vault-deposit]', { amount });
```

Comment:
can we add vault name too, just to future proof for later when we have more vaults

Reply:
also, why the brackets? is that some convention in posthog?

Reply:
Yes will add the vault name.

Bracket? I _thought_ it was convention, but think I was mistaken. I'll remove them.

Reply:
We can add vault address @0xalecks will that work?

Reply:
https://github.com/TempleDAO/temple/pull/486/commits/59ec0703e6bc95876708be100bdcdef4e74ceb5d